docs/provider-pkcs11.7

```diff
@@ -0,0 +1,250 @@
+.\" Automatically generated by Pandoc 2.14.0.3
+.\"
+.TH "provider-pkcs11" "7" "" "" "Configuration directives"
+.hy
+.SH NAME
+.PP
+pkcs11-provider - An OpenSSL provider that allows to directly interface
+with pkcs11 drivers.
+.SH DESCRIPTION
+.PP
+Starting with version 3.0 the OpenSSL project introduced a new modular
+system to extend OpenSSL that replaces the deprecated Engines modules.
+.PP
+Providers1 are loaded via configuration directives in the openssl
+configuration file (or directly loaded by applications).
+.PP
+The pkcs11 provider allows applications linked to openssl to use keys
+and cryptographic operations from a hardware or software token via their
+PKCS#112 driver and the use of pkcs11 URIs3.
```

Comment:
this is just a nit that the footnotes are rendered quite weirdly and make it look like PKCS#112, but probably nothing worth spending too much time with.